nixos/modules/services/network-filesystems/kubo.nix

```
@@ -278,6 +278,11 @@ in
             } \
           | ipfs --offline config replace -
       '';
+      postStop = mkIf cfg.autoMount ''
+        # After an unclean shutdown the fuse mounts at ${cfg.ipnsMountDir} and ${cfg.ipfsMountDir} are locked
+        umount ${cfg.ipnsMountDir} || true
+        umount ${cfg.ipfsMountDir} || true
```

```suggestion
        umount --quiet '${cfg.ipnsMountDir}' || true
        umount --quiet '${cfg.ipfsMountDir}' || true
```
I suggest adding `--quiet` to suppress "not mounted" error messages and to add quotes in case the paths contain spaces or other weird characters.